src/Uno.UI.RuntimeTests/Extensions/TimelineExtensions.cs

```diff
@@ -8,20 +8,11 @@
 
 namespace Uno.UI.RuntimeTests.Extensions;
 
-internal static class StoryboardExtensions
+internal static class TimelineExtensions
 {
-	public static void Begin(this Timeline timeline)
+	public static Storyboard Wrap(this Timeline timeline)
```

Comment:
Not sure about the naming of this one, sounds quite generic

```suggestion
	public static Storyboard ToStoryboard(this Timeline timeline)
```